From c57c223f7c9526b4eb3dfd38f9f352b7180aa808 Mon Sep 17 00:00:00 2001
From: Eric Sandeen <sandeen@redhat.com>
Date: Mon, 4 Aug 2014 19:01:47 -0400
Subject: [fs] btrfs: Cleanup the unused struct async_sched

Message-id: <1407179109-13714-221-git-send-email-sandeen@redhat.com>
Patchwork-id: 87132
O-Subject: [RHEL7.1 PATCH 220/422] btrfs: Cleanup the unused struct async_sched.
Bugzilla: 1071179
RH-Acked-by: Zach Brown <zab@redhat.com>

From: Qu Wenruo <quwenruo@cn.fujitsu.com>

commit f5961d41d7575faa6e2905daa08650aa388ba9d0
Author: Qu Wenruo <quwenruo@cn.fujitsu.com>
Date:   Fri Feb 28 10:46:02 2014 +0800

    btrfs: Cleanup the unused struct async_sched.

    The struct async_sched is not used by any codes and can be removed.

    Signed-off-by: Qu Wenruo <quwenruo@cn.fujitsu.com>
    Reviewed-by: Josef Bacik <jbacik@fusionio.com>
    Tested-by: David Sterba <dsterba@suse.cz>
    Signed-off-by: Josef Bacik <jbacik@fb.com>
Signed-off-by: Eric Sandeen <sandeen@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/btrfs/volumes.c b/fs/btrfs/volumes.c
index 3a1d88e..b358663 100644
--- a/fs/btrfs/volumes.c
+++ b/fs/btrfs/volumes.c
@@ -5323,13 +5323,6 @@ static void btrfs_end_bio(struct bio *bio, int err)
  }
 }
 
-struct async_sched {
- struct bio *bio;
- int rw;
- struct btrfs_fs_info *info;
- struct btrfs_work work;
-};
-
 /*
  * see run_scheduled_bios for a description of why bios are collected for
  * async submit.
-- 
1.7.1